src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,13 @@
-export const CommentList = () => (
-  <>Put the list here</>
-);
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  comments.length !== 0 ? (
```

Comment:
CommentList should just render a list of comments. Move this condition `comments.length !== 0 ?` to `PostInfo`